src/pytest_workflow/plugin.py

```diff
@@ -36,35 +39,46 @@ def pytest_collect_file(path, parent):
 class YamlFile(pytest.File):
     """This class collects YAML files and turns them into test items."""
 
+    def __init__(self, path, parent):
+        super(YamlFile, self).__init__(path, parent=parent)
```

Comment:
In python 3, the `super` call can be called without any arguments, like so:

```python
def __init__(self, path, parent):
    super().__init__(path, parent=parent)
```

Reply:
Thanks. I have removed this.